packages/storybook/stories/components/tooltip/documentation.mdx

```diff
@@ -1,36 +1,127 @@
-import { Canvas, Meta, Markdown } from '@storybook/blocks';
-import SpecificationsTooltip from '@ovhcloud/ods-components/src/components/tooltip/documentation/spec.md?raw';
+import { Canvas, Markdown, Meta } from '@storybook/blocks';
 import { Banner } from '../../banner';
-import { DocNavigator } from '../../doc-navigator';
 import * as TooltipStories from './tooltip.stories';
-import { LINK_ID } from '../../zeroheight';
+import { OdsIcon } from '@ovhcloud/ods-components/react';
+import { ODS_ICON_NAME } from '@ovhcloud/ods-components';
 
 <Meta of={ TooltipStories } name='Documentation' />
 
 <Banner of={ TooltipStories } />
 
-# Overview
-
-Tooltip component is used to display contextual information when the user hovers over or focus on an UI element in a page:
+_**Tooltip** component is used to display contextual information when the user hovers over or focus on a UI element in a page._
 
 <Canvas of={ TooltipStories.Overview } sourceState='none' />
 
-<DocNavigator of={ TooltipStories } linkId={ LINK_ID.TOOLTIP } />
+## Overview
+<div>
+  <div id='description'>
+    The **Tooltip** component is used to display brief, contextual information when users hover over or focus on an
```

Comment:
Tooltip component is used to display contextual information when the user hovers or focuses an element.